docs/outlook/add-and-remove-attachments-to-an-item-in-a-compose-form.md

```diff
@@ -45,82 +45,76 @@ You can attach a file to a message or appointment in a compose form by using the
-        // You can optionally pass any object that you would
-        // access in the callback function as an argument to
-        // the asyncContext parameter.
-        Office.context.mailbox.item.addFileAttachmentAsync(
-            `https://webserver/picture.png`,
-            'picture.png',
-            { asyncContext: null },
-            function (asyncResult) {
-                if (asyncResult.status === Office.AsyncResultStatus.Failed){
-                    write(asyncResult.error.message);
-                } else {
-                    // Get the ID of the attached file.
-                    const attachmentID = asyncResult.value;
-                    write('ID of added attachment: ' + attachmentID);
-                }
-            });
-    });
-}
-
-// Writes to a div with id='message' on the page.
-function write(message){
-    document.getElementById('message').innerText += message;
-}
+The following JavaScript example is a compose add-in that attaches a file, **picture.png**, from a web server to the message or appointment being composed. The callback function takes `asyncResult` as a parameter, checks for the result status, and gets the attachment ID if the method succeeds.
+
+```javascript
+// Add the specified file attachment to the item
+// being composed.
+// When the attachment finishes uploading, the
+// callback function is invoked and gets the attachment ID.
+// You can optionally pass any object that you would
+// access in the callback function as an argument to
+// the asyncContext parameter.
+Office.context.mailbox.item.addFileAttachmentAsync(
+    "https://webserver/picture.png",
+    "picture.png",
+    { asyncContext: { var1: 1, var2: 2 } },
+    (asyncResult) => {
+        if (asyncResult.status === Office.AsyncResultStatus.Failed) {
+            console.error(asyncResult.error.message);
+            return;
+        }
+
+        // Get the ID of the attached file.
+        const attachmentID = asyncResult.value;
+        console.log(`ID of added attachment: ${attachmentID}`);
+    }
+);
 ```
 
-To add an inline base64 image to the body of a message or appointment being composed, you must first get the current item body using the `Office.context.mailbox.item.body.getAsync` method before inserting the image using the `addFileAttachmentFromBase64Async` method. Otherwise, the image will not render in the body once it's inserted. For guidance, see the following JavaScript example, which adds an inline Base64 image to the beginning of an item body.
+To add an inline Base64 image to the body of a message or appointment being composed, you must first get the current item body using the `Office.context.mailbox.item.body.getAsync` method before inserting the image using the `addFileAttachmentFromBase64Async` method. Otherwise, the image won't render in the body once it's inserted. For guidance, see the following JavaScript example, which adds an inline Base64 image to the beginning of an item body.
```

Comment:
```suggestion
To add inline a Base64-encoded image to the body of a message or appointment being composed, you must first get the current item body using the `Office.context.mailbox.item.body.getAsync` method before inserting the image using the `addFileAttachmentFromBase64Async` method. Otherwise, the image won't render in the body once it's inserted. For guidance, see the following JavaScript example, which adds an inline Base64 image to the beginning of an item body.
```